src/geometry/geom2/applyTransforms.js

```diff
@@ -2,14 +2,14 @@ const mat4 = require('../../math/mat4')
 const vec2 = require('../../math/vec2')
 
 /**
- * Produces a canonicalized geometry by canonicalizing the sides.
- * Must be called before exposing any side data.
- * @param {geom2} geometry - the geometry to canonicalize
- * @returns {geom2} the given geometry with transformed points
+ * Apply the transforms of the given geometry.
+ * NOTE: Must be called before exposing any side data.
```

Comment:
I would change the note to something more clear like 'call this first if you need the side's data with transforms baked in'

Reply:
also perhaps mention, to sides ?